### PR TITLE
Use GitHub superlinter

### PR DESCRIPTION
**Pull request by bradfordb-vmware**

### CODE REVIEW

File: .github/linters/.markdown-lint.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#
+
+###############
+# Rules by id #
+###############
+MD004: false                  # Unordered list style
+MD007:
+  indent: 2                   # Unordered list indentation
+MD013:
+  line_length: 808            # Line length
```
Comment (xinzweb) [inline .github/linters/.markdown-lint.yml:25]: 808? Is that a typo?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ We’d love to accept your patches and contributions to this project. Please rev
 
 ## Contributor License Agreement
 
-All contributors to this project must have a signed Contributor License Agreement (**"CLA"**) on file with us. The CLA grants us the permissions we need to use and redistribute your contributions as part of the project; you or your employer retain the copyright to your contribution. Head over to https://cla.pivotal.io/ to see your current agreement(s) on file or to sign a new one.
+All contributors to this project must have a signed Contributor License Agreement (**"CLA"**) on file with us. The CLA grants us the permissions we need to use and redistribute your contributions as part of the project; you or your employer retain the copyright to your contribution. Head over to <https://cla.pivotal.io/> to see your current agreement(s) on file or to sign a new one.
```
Comment (xinzweb) [inline CONTRIBUTING.md:7]: Are we still `cla.pivotal.io`? Is there any vmware replacement?

File: .github/linters/.markdown-lint.yml (hunk 1 above)
Comment (bradfordb-vmware) [reply .github/linters/.markdown-lint.yml:25]: Nope -- this config file was copied from [github/super-linter](https://github.com/github/super-linter/blob/131055ece753ed219b22390d97e916244ec5f703/TEMPLATES/.markdown-lint.yml#L24-L25). I had to copy it to add a custom rule for MD024.

File: CONTRIBUTING.md (hunk 2 above)
Comment (bradfordb-vmware) [reply CONTRIBUTING.md:7]: This is the existing contributing guide and as far as I know we have not been given an updated one yet.

Comment (xinzweb) [reply CONTRIBUTING.md:7]: Wow, that's very long to start with. If we need a different length, we can revisit later.

File: .github/linters/.markdown-lint.yml (hunk 1 above)
Comment (xinzweb) [reply .github/linters/.markdown-lint.yml:25]: That's long. Let's revisit if needed.

**Pull request merged**